src/app/data/hierarchical-data.ts

```diff
@@ -519,17 +543,26 @@ export const CUSTOMERS = [{
             ProductID: 22,
             UnitPrice: 16.8000,
             Quantity: 6,
-            Discount: 5.0000001e-002
+            Discount: 5.0000001e-002,
+            Weight: 2.6,
+            Length: 26.8,
+            TotalPrice: 226.50
         }, {
             ProductID: 57,
             UnitPrice: 15.6000,
             Quantity: 15,
-            Discount: 5.0000001e-002
+            Discount: 5.0000001e-002,
+            Weight: 3.1,
+            Length: 24.3,
+            TotalPrice: 219.25
         }, {
             ProductID: 65,
             UnitPrice: 16.8000,
             Quantity: 20,
-            Discount: 0.0000000e+000
+            Discount: 0.0000000e+000,
```

Comment:
Why setting the discount in this format, seems curious

Reply:
This is not a change applied by me I only added the properties below it